python-avd/pyavd/_eos_designs/schema/schema_fragments/sflow_settings.schema.yml

```diff
@@ -64,6 +64,26 @@ keys:
               convert_types:
                 - int
               default: use_default_mgmt_method_vrf
+      export_to_cloudvision:
+        type: dict
+        description: sFlow will be configured if either `destinations` or `export_to_cloudvision.enabled` is set. One of them is required.
+        keys:
+          enabled:
+            type: bool
```

Comment:
add a description stating that when this is enabled, AVD will automatically configure a destination to 127.0.0.1 with port 6343 in the VRF corresponding to `use_default_mgmt_method_vrf`.